### DISCUSSION TITLE
<Slackes/ Club.cc / slackes.me Is a scammer!>

### DISCUSSION THREAD

**Discussion by JayzFrost** [community/community#45431] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

https://imgur.com/a/xTb77RS

I bought a project from this guy last 1/23/2023 hoping that he is legit. 

Bought eft 150 private project + 50$ reseller panel.
He ended up blocking me after delaying it for days. Deleted his messages. He is actively scamming people through this website and pretending to be a dev. got blocked and banned from his discord.

https://github.com/Slackes   < Github Profile
Slackes#0001 < Discord user
[REDACTED DUE TO MEMBER SAFETY]   < Discord server invite

**Comment by queenofcorgis** [comment, marked as answer]
Hi @JayzFrost 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
